src/utilities/aspect-ratio/demo/image.twig

```diff
@@ -0,0 +1,17 @@
+{% if aspectRatio %}
+  {% set style = (style|default('')) ~ ';--aspect-ratio:' ~ aspectRatio %}
+{% endif %}
+
+<img
+  class="{{ aspectRatioClass }}"
+  src="{{ src }}"
+  alt=""
+  width="900"
```

Comment:
What happens now if `width` isn't set?

Reply:
I adjusted the CSS and removed this `width` attribute. The intent is that it should work without it.